frontend/src/lib/components/appLogo/AppLogo.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+  import {page} from '$app/stores';
+  import type {AppLogoProps} from './AppLogo.type';
+
+  type $$Props = AppLogoProps;
+  export let alt: $$Props['alt'] = $page.data.appLabels.viewTexts.publisherName;
```

Comment:
The reason I'm including `alt` is that the logo could be the only visible indication of the publisher of the app and it may be used outside the navigation menu as well. In contrast to icons, it's not purely decorative so IMO it should be visible to screen readers as well.

But if you feel strongly that `alt` should  be removed, I'll do that.